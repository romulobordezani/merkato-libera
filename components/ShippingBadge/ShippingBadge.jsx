import React from 'react';
import PropTypes from 'prop-types';

import styles from './ShippingBadge.module.scss';

const ShippingBadge = ({ item }) => {
  const { freeShipping } = item;

  if (!freeShipping) {
    return null;
  }

  return (
    <span className={styles.ShippingBadge}>
      <img src="/images/ic_shipping.png" alt="Free Shipping" />
    </span>
  );
};

ShippingBadge.propTypes = {
  item: PropTypes.object.isRequired
};

export default ShippingBadge;
