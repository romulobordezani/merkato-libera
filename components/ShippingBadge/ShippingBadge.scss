.ShippingBadge {
  outline: none;
  margin-left: 6px;
}
