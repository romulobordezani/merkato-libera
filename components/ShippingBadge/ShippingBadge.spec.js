import React from 'react';
import renderer from 'react-test-renderer';

import ShippingBadge from './ShippingBadge';

const ItemMock = require('../../__mocks__/Item');

describe('ShippingBadge', () => {
  it('Snapshot renders default', () => {
    expect(renderer.create(<ShippingBadge item={ItemMock} />)).toMatchSnapshot();
  });
});
