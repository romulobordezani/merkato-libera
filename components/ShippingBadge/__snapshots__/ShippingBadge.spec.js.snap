// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingBadge Snapshot renders default 1`] = `null`;
